socialdistribution/views.py

```diff
@@ -32,6 +32,10 @@ def get_server_to_endpoints_mapping(self) -> list[tuple[Server, list[str]]]:
         server_endpoints_tuples = []
         for server in Server.objects.all():
             resp = server.get('/authors/')
+            if resp.status_code != 200:
+                resp = server.get('/authors')
+                if resp.status_code != 200:
+                    continue
```

Comment:
```suggestion
            resp = server.get('/authors')
```
This also works just fine for our current endpoints so let's do this to be consistent with the follow/views.py